### PR TITLE
added rtoml package

### PR DESCRIPTION
**Pull request by AhmedAliStats**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
Hi @AhmedAliStats, thanks for the recipe

There is still no conda forge package available for https://pypi.org/project/setuptools-rust/, a recipe for that would need to be created.

@

**Comment by AhmedAliStats** [comment]
> Hi @AhmedAliStats, thanks for the recipe
> 
> There is still no conda forge package available for https://pypi.org/project/setuptools-rust/, a recipe for that would need to be created.
> 
> @

@goanpeca 

Thank you very much for your reply.

I think setuptools-rust should already be on conda-forge but I don't know why it's not being picked up by mamba.
https://github.com/conda-forge/setuptools-rust-feedstock

**Comment by goanpeca** [comment]
@AhmedAliStats you are right, my bad.
 You need to use a hyphen.

File: recipes/rtoml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip >=21.0.1
+    - python >=3.7
+    - setuptools_rust
```
Comment (goanpeca) [inline recipes/rtoml/meta.yaml:21]:
```suggestion
    - setuptools-rust
```

Comment (AhmedAliStats) [reply recipes/rtoml/meta.yaml]: @goanpeca Thank you very much for helping with this.

File: recipes/rtoml/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rtoml-{{ version }}.tar.gz
+  sha256: 6d55fa9df88256c6762f01f18b8bd3f6dc1836600ea8655eb3719a2b8f3df764
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
```
Comment (goanpeca) [inline recipes/rtoml/meta.yaml:18]:
```suggestion
  build:
    - {{ compiler('rust') }}
  host:
```

Comment (goanpeca) [reply recipes/rtoml/meta.yaml:18]: We forgot about the rust compiler :)

**Comment by AhmedAliStats** [comment]
@conda-forge-admin, please ping team

**Review by goanpeca — APPROVED** [review]
Hi @AhmedAliStats looks great. I just want to confirm with someone from the team

@conda-forge/help-python

File: recipes/rtoml/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/samuelcolvin/rtoml
+  summary: A better TOML library for python implemented in rust.
+  license: MIT
+  license_file: LICENSE
```
Comment (BastianZim) [inline recipes/rtoml/meta.yaml:40]: You probably need to include the Rust licenses as well: https://conda-forge.org/docs/maintainer/adding_pkgs.html#licenses-of-included-dependencies

File: recipes/rtoml/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+build:
+  script: 
+  - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+  - {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - libgcc-ng  # [linux]
```
Comment (BastianZim) [inline recipes/rtoml/meta.yaml:21]: This is normally included in a compiler. Are you sure you only need that?

**Pull request closed without merging**